### PR TITLE
Merge-able open dropdown on input focus

### PR DESCRIPTION
**Pull request by yidea**
Closed previous PR #7 which is not merge-able to the upstream/master , instead create this PR with the same change but merge-able with upstream/master and fixed failed unit test 

Changes: 
- Same changes copied from https://github.com/walmartlabs/typeahead.js/pull/7
- Make sure unit test don't fail on both browser and phantomjs (there's a bug of comparing html content)

/cc @ryan-roemer @alexlande

### CODE REVIEW

File: dist/typeahead.js [hunk 1]
```diff
@@ -1015,7 +1015,7 @@
...
+            }).on("blured", this._closeDropdown).on("blured", this._setInputValueToQuery).on("enterKeyed tabKeyed", this._handleSelection).on("queryChanged", this._clearHint).on("queryChanged", this._getSuggestions).on("queryChangedEmpty", this._clearSuggestions).on("whitespaceChanged", this._updateHint).on("queryChanged whitespaceChanged", this._openDropdown).on("queryChanged whitespaceChanged", this._setLanguageDirection).on("escKeyed", this._closeDropdown).on("escKeyed", this._setInputValueToQuery).on("tabKeyed upKeyed downKeyed", this._managePreventDefault).on("upKeyed downKeyed", this._moveDropdownCursor).on("upKeyed downKeyed", this._openDropdown).on("tabKeyed leftKeyed rightKeyed", this._autocomplete);
```
Comment (alexlande) [inline dist/typeahead.js]: Fix `blurred` typo again.

**Comment by alexlande** [comment]
LGTM except for the `blured` typo that snuck back in. Make sure to run tests again after updating that.

**Comment by ryan-roemer** [comment]
JS LGTM. Approved after tests pass. (And ping me to merge if I still need to).

**Comment by yidea** [comment]
@ryan-roemer test passed, I still don't have the right permission, can u help me merge this.

**Comment by ryan-roemer** [comment]
@yidea -- You're good to go!

**Pull request merged**